### PR TITLE
Updating licenses, access, and RBAC

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

- Updates RBAC links
- Removes stale page
- Updates permissions specific links (they just go to the main page anyway)
- Cut back on the service tokens permissions list as it was inconsistent and linking to areas that don't exist. Updated to link to the permissions page at the top

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/best-practices/how-we-mesh/mesh-5-faqs
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/about-cloud/architecture
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/manage-access/about-access
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/manage-access/cloud-seats-and-users
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/manage-access/enterprise-permissions
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/manage-access/environment-permissions
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/manage-access/licenses-and-groups
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/manage-access/set-up-sso-google-workspace
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/cloud/manage-access/set-up-sso-okta
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/dbt-cloud-apis/service-tokens
- https://docs-getdbt-com-git-cleanup-access-dbt-labs.vercel.app/docs/deploy/webhooks

### CODE REVIEW

File: website/docs/docs/cloud/manage-access/enterprise-permissions.md [hunk 1]
```diff
@@ -22,22 +22,14 @@ The following roles and permission sets are available for assignment in dbt Clou
 
 :::tip Licenses or Permission sets
 
-The user's [license](/docs/cloud/manage-access/seats-and-users) type always overrides their assigned permission set. This means that even if a user belongs to a dbt Cloud group with 'Account Admin' permissions, having a 'Read-Only' license would still prevent them from performing administrative actions on the account.
+The user's [license](/docs/cloud/manage-access/about-access) type always overrides their assigned permission set. This means that even if a user belongs to a dbt Cloud group with 'Account Admin' permissions, having a 'Read-Only' license would still prevent them from performing administrative actions on the account.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/enterprise-permissions.md]:
```suggestion
The user's [license](/docs/cloud/manage-access/about-user-access) type always overrides their assigned permission set. This means that even if a user belongs to a dbt Cloud group with 'Account Admin' permissions, having a 'Read-Only' license would still prevent them from performing administrative actions on the account.
```

File: website/docs/best-practices/how-we-mesh/mesh-5-faqs.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ There’s model-level access within dbt, role-based access for users and groups
...
 
-[dbt Cloud Enterprise plans](https://www.getdbt.com/pricing) support [role-based access control (RBAC)](/docs/cloud/manage-access/enterprise-permissions#how-to-set-up-rbac-groups-in-dbt-cloud) that manages granular permissions for users and user groups. You can control which users can see or edit all aspects of a dbt Cloud project. A user’s access to dbt Cloud projects also determines whether they can “explore” that project in detail. Roles, users, and groups are defined within the dbt Cloud application via the UI or by integrating with an identity provider.
+[dbt Cloud Enterprise plans](https://www.getdbt.com/pricing) support [role-based access control (RBAC)](/docs/cloud/manage-access/about-user-access#role-based-access-control-) that manages granular permissions for users and user groups. You can control which users can see or edit all aspects of a dbt Cloud project. A user’s access to dbt Cloud projects also determines whether they can “explore” that project in detail. Roles, users, and groups are defined within the dbt Cloud application via the UI or by integrating with an identity provider.
```
Comment (runleonarun) [inline website/docs/best-practices/how-we-mesh/mesh-5-faqs.md:218]:
```suggestion
[dbt Cloud Enterprise plans](https://www.getdbt.com/pricing) support [role-based access control (RBAC)](/docs/cloud/manage-access/about-access#role-based-access-control-) that manages granular permissions for users and user groups. You can control which users can see or edit all aspects of a dbt Cloud project. A user’s access to dbt Cloud projects also determines whether they can “explore” that project in detail. Roles, users, and groups are defined within the dbt Cloud application via the UI or by integrating with an identity provider.
```

File: website/docs/best-practices/how-we-mesh/mesh-5-faqs.md [hunk 3]
```diff
@@ -215,7 +215,7 @@ There’s model-level access within dbt, role-based access for users and groups
...
 
-[dbt Cloud Enterprise plans](https://www.getdbt.com/pricing) support [role-based access control (RBAC)](/docs/cloud/manage-access/enterprise-permissions#how-to-set-up-rbac-groups-in-dbt-cloud) that manages granular permissions for users and user groups. You can control which users can see or edit all aspects of a dbt Cloud project. A user’s access to dbt Cloud projects also determines whether they can “explore” that project in detail. Roles, users, and groups are defined within the dbt Cloud application via the UI or by integrating with an identity provider.
+[dbt Cloud Enterprise plans](https://www.getdbt.com/pricing) support [role-based access control (RBAC)](/docs/cloud/manage-access/about-access#role-based-access-control-) that manages granular permissions for users and user groups. You can control which users can see or edit all aspects of a dbt Cloud project. A user’s access to dbt Cloud projects also determines whether they can “explore” that project in detail. Roles, users, and groups are defined within the dbt Cloud application via the UI or by integrating with an identity provider.
```
Comment (matthewshaver) [inline website/docs/best-practices/how-we-mesh/mesh-5-faqs.md]:
```suggestion
[dbt Cloud Enterprise plans](https://www.getdbt.com/pricing) support [role-based access control (RBAC)](/docs/cloud/manage-access/about-user-access#role-based-access-control-) that manages granular permissions for users and user groups. You can control which users can see or edit all aspects of a dbt Cloud project. A user’s access to dbt Cloud projects also determines whether they can “explore” that project in detail. Roles, users, and groups are defined within the dbt Cloud application via the UI or by integrating with an identity provider.
```

File: website/docs/docs/dbt-cloud-apis/service-tokens.md [hunk 4]
```diff
@@ -36,80 +36,36 @@ You can assign service account tokens to any permission set available in dbt Clo
 
 ### Team plans using service account tokens
 
-The following permissions can be assigned to a service account token on a Team plan.
+The following permissions can be assigned to a service account token on a Team plan. See the [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions) article for more information about what these roles are able to do.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/service-tokens.md]: Suggest using "refer to" and simplifying the sentence.
```suggestion
The following permissions can be assigned to a service account token on a Team plan. Refer to [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions) for more information about these roles. 
```

File: website/docs/docs/dbt-cloud-apis/service-tokens.md [hunk 5]
```diff
@@ -36,80 +36,36 @@ You can assign service account tokens to any permission set available in dbt Clo
...
+- Job Admin 
+- Job Runner
+- Metadata Only
+- Member
+- Read-only 
+- Semantic Layer Only 
 
 ### Enterprise plans using service account tokens
 
 The following permissions can be assigned to a service account token on an Enterprise plan. For more details about these permissions, see "[Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions)."
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/service-tokens.md]: Suggest making this consistent with above wording
```suggestion
Refer to [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions) for more information about these roles. 
```

**Review by runleonarun — APPROVED** [review]
Looks good with two non blocking suggestions

**Pull request merged**